apis/server/network_bridge.go

```diff
@@ -77,6 +77,22 @@ func (s *Server) deleteNetwork(ctx context.Context, rw http.ResponseWriter, req
 	return nil
 }
 
+func (s *Server) disconnectNetwork(ctx context.Context, rw http.ResponseWriter, req *http.Request) error {
+	disconnect := &types.NetworkDisconnect{}
+	// decode request body
+	if err := json.NewDecoder(req.Body).Decode(disconnect); err != nil {
+		return httputils.NewHTTPError(err, http.StatusBadRequest)
+	}
+	// validate request body
+	if err := disconnect.Validate(strfmt.NewFormats()); err != nil {
+		return httputils.NewHTTPError(err, http.StatusBadRequest)
+	}
+
+	name := mux.Vars(req)["name"]
```

Comment:
I sugget using `network` with the naming, since it will be some kind of clear. 😃 Not blocking